From b435a8dc923d9a7a03493b0ac787a08a18007a09 Mon Sep 17 00:00:00 2001
From: padmashree mandri <padmashree.mandri@intel.com>
Date: Wed, 16 Oct 2024 09:58:43 +0000
Subject: [PATCH] Enable capture capapbilities

This patch adds permission of capture audio output,
which is required by echo-reference device to capture
output for looopback purpose.

Signed-off-by: padmashree mandri <padmashree.mandri@intel.com>
---
 tests/EmbeddedKitchenSinkApp/AndroidManifest.xml | 1 +
 1 file changed, 1 insertion(+)

diff --git a/tests/EmbeddedKitchenSinkApp/AndroidManifest.xml b/tests/EmbeddedKitchenSinkApp/AndroidManifest.xml
index 695678f996..bfbf9a621a 100644
--- a/tests/EmbeddedKitchenSinkApp/AndroidManifest.xml
+++ b/tests/EmbeddedKitchenSinkApp/AndroidManifest.xml
@@ -127,6 +127,7 @@
     <uses-permission android:name="android.permission.MODIFY_AUDIO_ROUTING"/>
     <uses-permission android:name="android.permission.MODIFY_AUDIO_SETTINGS"/>
     <uses-permission android:name="android.permission.RECORD_AUDIO" />
+    <uses-permission android:name="android.permission.CAPTURE_AUDIO_OUTPUT" />
     <uses-permission android:name="android.permission.READ_PRIVILEGED_PHONE_STATE"/>
     <uses-permission android:name="android.permission.MODIFY_DAY_NIGHT_MODE"/>
     <uses-permission android:name="android.permission.MODIFY_PHONE_STATE"/>
-- 
2.34.1

